README.md

```diff
@@ -89,6 +89,8 @@ If you can't find your distro here, you will have to [build from source](#buildi
 
 If you create a package for any other distribution, please consider contributing the template.
 
+If you are using **Debian** (or other Debian derivatives), you can install [polybar](https://tracker.debian.org/pkg/polybar) using `sudo apt install polybar`.
```

Comment:
@patrick96 It's not yet in the derivatives, it will only downstream to their new releases, so you're right, for example the first Ubuntu release which will have polybar is 20.10 (20.04 won't), the same goes for Linux Mint or other derivatives.

@utkarsh2102 I believe it's also worth calling out that backports needs to be used to install the package on Buster/current stable.

Maybe the wording about the derivatives should mention something about the package being downstreamed from Debian and that it only new releases will contain the package.

Packages can also be downloaded from https://packages.debian.org/buster-backports/polybar  (download link at the bottom of page) and there should be a version which will work for all current released derivatives (binary for buster-backports should work for all current ones)

Reply:
Hi,
I guess, yes. I could add about "other derivatives" once they're available in Ubuntu, Mint, etcetera.
And I'll add a line about installation from buster/stable.